### DISCUSSION TITLE
I crash on inject

### DISCUSSION THREAD

**Discussion by NothingToSay1233** [YimMenu/YimMenu#2247] [Q&A]
I crash while I try to inject.
I look in cout.log and it says
[WARN/batch.hpp:113] Failed to find 'PD'.

**Comment by xiaoxiao921** [comment]
Duplicate of https://github.com/YimMenu/YimMenu/issues/2242

**Reply by yuaqn** [reply]
wdym, also i have the same problem

**Reply by Yimura** [reply]
> wdym, also i have the same problem

Did you update YimMenu.dll?

**Reply by yuaqn** [reply]
oh no i use tessioscript

**Reply by yuaqn** [reply]
and everytime i try to inject that error pops up and my game crashes
:(

**Reply by Yimura** [reply]
> oh no i use tessioscript

I can't seem to remember where you said a Lua script was crashing and not something else in YimMenu...

**Reply by yuaqn** [reply]
ima download yim menu and tell u if the issue is still there gimne a min

**Reply by yuaqn** [reply]
i downloaded yim menu dll but idk how to inject

**Reply by Yimura** [reply]
> i downloaded yim menu dll but idk how to inject

Have you tried looking for a solution by yourself?

**Reply by yuaqn** [reply]
yeah but nothing works

**Reply by Yimura** [reply]
I sincerely doubt you manage to single out all possible solutions within a few minutes and came to the conclusion that nothing works.

Kindly use whatever media works the best for you Googling or by using YouTube. Try to do some search and learning yourself before you get the solution given to you.

**Reply by yuaqn** [reply]
i did it and that PD is still not found

**Reply by yuaqn** [reply]
okay ill do that thanks

**Reply by Yimura** [reply]
> i did it and that PD is still not found

Probably forgot to download the latest YimMenu.dll from the releases page then.
